pcsx2/Memory.h

```diff
@@ -154,14 +154,14 @@ namespace SysMemory
 #define psHu64(mem) (*(u64*)&eeHw[(mem)&0xffff])
 #define psHu128(mem) (*(u128*)&eeHw[(mem)&0xffff])
 
-#define psMs8(mem) (*(s8*)&eeMem->Main[(mem)&0x1ffffff])
-#define psMs16(mem) (*(s16*)&eeMem->Main[(mem)&0x1ffffff])
-#define psMs32(mem) (*(s32*)&eeMem->Main[(mem)&0x1ffffff])
-#define psMs64(mem) (*(s64*)&eeMem->Main[(mem)&0x1ffffff])
-#define psMu8(mem) (*(u8*)&eeMem->Main[(mem)&0x1ffffff])
-#define psMu16(mem) (*(u16*)&eeMem->Main[(mem)&0x1ffffff])
-#define psMu32(mem) (*(u32*)&eeMem->Main[(mem)&0x1ffffff])
-#define psMu64(mem) (*(u64*)&eeMem->Main[(mem)&0x1ffffff])
+#define psMs8(mem) (*(s8*)&eeMem->Main[(mem)&(CHECK_EXTRAMEM?0x7ffffff:0x1ffffff)])
```

Comment:
Not sure if these are even used, they might be able to be removed. I'll check later.

Reply:
They are indeed not used currently.

Reply:
On closer inspection, the only `ps*` macros from this header that seem to be used are:
* `PSM`
* `psHu8`
* `psHu16`
* `psHu32`

Reply:
May as well purge them then. psHu is for hardware registers, so they're unaffected by this change. Probably keep the psHu64/128 just for completeness's sake.

Reply:
Did that just now. It seems `psSu32` and `psSu128` are also used, so I kept the `psSu64` macro as well.